metadata-ingestion/src/datahub/ingestion/source/feast.py

```diff
@@ -296,12 +330,29 @@ def _get_feature_view_workunit(self, feature_view: FeatureView) -> MetadataWorkU
 
         feature_view_name = f"{self.feature_store.project}.{feature_view.name}"
 
+        aspects = [
+            BrowsePathsClass(paths=[f"/feast/{self.feature_store.project}"]),
+            StatusClass(removed=False),
+        ]
+
+        if feature_view.tags.get("name"):
+            tag = feature_view.tags.get("name")
+            tag_association = TagAssociationClass(tag=builder.make_tag_urn(tag))
+            global_tags_aspect = GlobalTagsClass(tags=[tag_association])
+            aspects.append(global_tags_aspect)
+
+        if feature_view.owner:
```

Comment:
Just note: If you've attached owners in DataHub UI, this will replace them by default :)

Reply:
See comment above